### PR TITLE
Implemented Delete Ability for Users and Admins

### LINKED ISSUE #78: Add "Straw" as a color option on the far light side of appearance scale
Appearance slider is missing "Straw" (SRM 2-5) as an option on the light side of the scale.

Objectives:
1. Add "Straw" as a new color on scoresheet form, at the far left
2. Add "Straw" as a new color on pdf template, at the far left

### LINKED ISSUE #137: Ability for admins / users to delete scoresheets
Sometimes, users accidentally create extra scoresheets. Having a delete functionality is critical

Front End Requirements:
* Can only delete scoresheets on in-progress flights
* Delete confirmation
* Admins should be able to recover soft-deleted scoresheets from admin panel

Back End Requirements:
* Soft delete - scoresheets are flagged as 'deleted' for 24 hours before permanent db removal
* Users should be blocked from deleting scoresheets they did not create

### LINKED ISSUE #138: Ability for admins / users to delete empty flights
Sometimes, users accidentally create extra flights. Having a delete functionality is critical to app flow and self-service.

Front End Requirements:
* Can only delete flights with no scoresheets
* Delete confirmation

Back End Requirements:
* Users should be blocked from deleting flights they did not create

### LINKED ISSUE #150: Slider misclick bug
* Many users have reported issues with "random" slider click triggers when clicking outside their bounds
* This is due to rotated labels having a much larger bound

### PR DESCRIPTION
**Pull request by prphntm63**
* Resolved #137 
* Resolved #138 
* Fixed #150 
* Resolved #78 
* Fixed various other minor bugs

### CODE REVIEW

File: controllers/ScoresheetController.js [hunk 1]
```diff
@@ -126,6 +127,35 @@ scoresheetController.getScoresheetData = function (req, res) {
   });
 };
 
+scoresheetController.deleteScoresheet = function (req, res) {
```
Comment (PH-Lazurous) [inline controllers/ScoresheetController.js:130]: Do we straight delete? Or add a deleted field? Or simply move it to an archived table? The only thing that scares me on this is admin abuse, with a flag or a copy or if somewhere we can recover.

File: controllers/FlightController.js [hunk 2]
```diff
@@ -150,26 +147,29 @@ flightController.submitFlight = function (req, res) {
     })
     .catch((err) => {
       jsonErrorProcessor(err, res);
-      console.log(err);
     });
 };
```
Comment (PH-Lazurous) [inline controllers/FlightController.js:152]: Repeat comment ... Do we straight delete? Or add a deleted field? Or simply move it to an archived table? The only thing that scares me on this is admin abuse, with a flag or a copy or if somewhere we can recover.

Comment (PH-Lazurous) [reply controllers/FlightController.js:152]: Not sure why it marked this comment here ... NOT where i clicked ...

File: controllers/ScoresheetController.js (hunk 1 above)
Comment (prphntm63) [reply controllers/ScoresheetController.js:130]: We should be set up to be doing paranoid deletes

**Pull request merged**